doc/source/workflow.rst

```diff
@@ -0,0 +1,726 @@
+replaced by the "regular" distribution created initrd after the initial boot of
+the image.
+
+The following list are some of the variables that might included as kernel
+parameters:
+
+``BOOTIF``
+  This variable sets the interface to boot in *PXE* images. Its is the MAC
+  address of the desired interface. If ``BOOTIF`` is not set the boot code
+  selects the first interface responding to the DHCP server.
+
+``DEVICE_TIMEOUT``
+  This variable sets the time (in seconds) that the system waits until a
+  storage device is considered to be unaccessible. By default this value is set
+  to 60 seconds. It can be handy in order to limit the wait time, specially
+  if there are operations pending on removable devices which may not be always
+  present.
+
+``DIALOG_LANG``
+  This variable holds the language code that will be used to select the
+  corresponding language literals used in the installation dialogs. By default
+  is set to 'ask', it meaning the user is prompted to select one. If unattended
+  mode has been selected, ``DIALOG_LANG`` is always set to US English.
+
+``KIWI_ALLOW_HOOK_CMD_hookname``
+  This variable enables the execution of a command right after the execution of
+  the corresponding hook script identified by *hookname*. If set to ``1`` the
+  post command related to *hookname* will be executed. 
+  
+``kiwi_cowdevice``
+  This variable sets the device to use for copy-on-write (cow) operations
+  in hybrid images. If used, ``kiwi_cowsystem`` also needs to be defined.
+
+``kiwi_cowsystem``
+  This variable sets the cow file to use inside the write partition defined by
+  ``kiwi_device`` in hybrid images. If used, ``kiwi_cowdevice`` also needs to
+  be defined.
+
+``kiwidebug``
+  This variable sets to enter on a limited shell in case there is a fatal error
+  during the boot process. The default behavior is to reboot after 120 seconds,
+  this variable prevents this behavior setting its value to ``1``.
+
+``KIWI_FORBID_HOOKS``
+  This variable disables all post commands int he boot process.
+
+``KIWI_HOOK_CMD_hookname``
+  This variable sets the post command of the corresponding hook script
+  identified by *hookname*. The value must be the command line including
+  all arguments all in a single string.
+
+``kiwi_hybridpersistent``
+  This variable sets if a write partition for hybrid images should be created or
+  not. It is boolean value represented by 'true' or 'false'.
+
+``kiwi_hybridpersistent_cow_filename``
+  This variable sets the filename of the cow file for hybrid images. By default
+  it is set to ``Live OS's persistent storage.fs``.
+
+``kiwi_hybridpersistent_filesystem``
+  This variable sets the filesystem to be used within the write partition of
+  hybrid images. By default is set to btrfs.
+
+``kiwi_ramonly``
+  This variable sets to mount the clicfs or unionfs read/write device on a
+  ram disk or not. Any empty value activates the ramonly mode.
+
+``LOCAL_BOOT``
```

Comment:
@schaefi Do you think it makes sense to include ``$root`` and ``$disk`` together with the`` $LOCAL_BOOT``? Maybe the usage of those, including LOCAL_BOOT should be documented as a debugging strategy in _Boot Debugging_ section like ``kiwidebug`` or the _dropbear_ service.